tests/follow-data-only/copydb.sh

```diff
@@ -83,3 +83,18 @@ psql -d ${PGCOPYDB_SOURCE_PGURI} -c "${sql}" > /tmp/s.out
 psql -d ${PGCOPYDB_TARGET_PGURI} -c "${sql}" > /tmp/t.out
 
 diff /tmp/s.out /tmp/t.out
+
+# count the number of rows in the generated_column_test table
+# on the source and target databases
+sql="select count(*) from generated_column_test"
+src_count=`psql -AtqX -d ${PGCOPYDB_TARGET_PGURI} -c "${sql}"`
+tgt_count=`psql -AtqX -d ${PGCOPYDB_SOURCE_PGURI} -c "${sql}"`
+test ${src_count} -gt 0
+test ${tgt_count} -gt 0
+test ${src_count} -eq ${tgt_count}
```

Comment:
Can we use output files and then `diff`? When the test fail, it makes it easier to debug.